internal/tools/aws/database.go

```diff
@@ -47,7 +47,7 @@ func (d *RDSDatabase) Provision(store model.InstallationDatabaseStoreInterface,
 }
 
 // Teardown removes all AWS resources related to a RDS database.
-func (d *RDSDatabase) Teardown(keepData bool, logger log.FieldLogger) error {
+func (d *RDSDatabase) Teardown(store model.InstallationDatabaseStoreInterface, keepData bool, logger log.FieldLogger) error {
```

Comment:
0/5 nitpick: is the name `InstallationDatabaseStoreInterface` redundant? Maybe just `InstallationDatabaseInterface` is better? The `Store` seems extraneous to me, or maybe even confusing because we refer to s3/minio as the "filestore".

Reply:
I agree but given that this an old interface code, I'm not refactoring it in this PR. I will put a comment for followup changes.

Reply:
Oh sorry, I thought it was a new interface. Definitely a comment is enough, or just ignore my feedback LOL